contents/blog/why-infra-is-our-competitive-advantage.md

```diff
@@ -0,0 +1,45 @@
+---
+date: 2022-05-27
+title: Why infrastructure is a competitive advantage for us
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog"]
+featuredImage: ../images/blog/migrating-hog.png
+featuredImageType: full
+---
+
+We're hiring more SREs. I think it's worth explaining why I think PostHog is the most exciting place to be an SRE right now. 
+
+## Building a product
+
+In most places, with the exception of outages, working in infrastructure means you're working on an invisible part of the company.
+Sure, building out fast CI pipelines and continuous deployment makes the rest of the engineering team more productive, but all your competitors do this too.
```

Comment:
I would not limit this to CI/CD. The overall goal is to improve the reliability of systems as well as  empower other engineers in the company to do more (providing methodologies, tools, etc..)

Reply:
i've updated the post to reflect this